dealership/contracts.py

```diff
@@ -1,12 +1,75 @@
+from dealership.customers import Customer, Employee
+from dealership.vehicles import Car, Truck, Motorcycle
+
 class Contract(object):
-    pass
+    '''Represents a contract with the dealership. Must be implemented as a
```

Comment:
I really like your use of docstrings throughout the assignment.  It's definitely something I need to get better about.
Although being super nitpicky- PEP8 specifies triple double quotes for docstrings.